#all
lookat_joint_names: [lookat_lin_joint, lookat_x_joint, lookat_y_joint, lookat_z_joint]
chain_joint_names: [head_powerball_bottom_joint, head_powerball_side_joint]

# lookat_driver
update_rate: 10.0

# lookat controller parameters
base_link: head_powerball_base_link
tip_link: lookat_focus_frame
chain_vel_pub_topic: /head_controller/command_vel
ptp_vel: 0.3


# lookat frame follower
focus_goal: focus_goal
